sample/src/main/java/com/smileidentity/sample/compose/components/SmileConfigModalBottomSheet.kt

```diff
@@ -37,6 +39,8 @@ fun SmileConfigModalBottomSheet(
     dismissable: Boolean = true,
 ) {
     var configInput by remember { mutableStateOf("") }
+    val context = LocalContext.current
+    val scanner = GmsBarcodeScanning.getClient(context)
```

Comment:
Does this need to be wrapped in a `remember`? Something like
```suggestion
    val scanner by remember(context) { GmsBarcodeScanning.getClient(context) }
```